README.md

```diff
@@ -1,18 +1,38 @@
 # Managed Structure Template
-This template is a starting point for creating programs that can run as Griptape Cloud Managed Structures. 
+
+This template is a starting point for creating programs that can run as Griptape Cloud Managed Structures.
 It provides a basic project layout for creating a Managed Structure, but you can customize it to fit your needs.
 
 See the [Griptape CLI](https://github.com/griptape-ai/griptape-cli?tab=readme-ov-file#skatepark-emulator) instructions for how to get started with this template.
 
 ## Project Structure
+
 The template provides the following project layout:
 
 - `structure.py` Contains the Managed Structure code.
 - `requirements.txt` Contains the dependencies for the Managed Structure. These will be automatically installed when the Structure is registered with Skatepark.
-- `example-client/client.py` Contains an example client that _uses_ the Managed Structure's API. This is useful for testing your Managed Structure locally but ultimately you will want to integrate your Managed Structure with your own application. 
+- `structure_file.yaml` Contains the Managed Structure configuration required by Skatepark and Griptape Cloud to properly build and run the Structure.
+- `example-client/client.py` Contains an example client that _uses_ the Managed Structure's API. This is useful for testing your Managed Structure locally but ultimately you will want to integrate your Managed Structure with your own application.
 - `example-client/pyproject.toml` Contains the dependencies for the example client. Poetry is only used for the example client and is not used by the Managed Structure itself.
 
+### Structure File Contents
+
+Griptape Cloud and Skatepark make use of the `structure_file.yaml` for understanding the build-time and run-time configuration of the Structure.
+
+The contents of the configuration file are as follows:
+
+```yaml
+version: 1.0 # Defines the version of the structure_file to use
+runtime: python3 # Defines the runtime environment for the Structure
+runtime_version: 3.11 # Defines the specific version of the runtime environment for the Structure
+build: # Defines the build-time configuration for the Structure
+  requirements_file: requirements.txt # Defines the path to the requirements.txt file for the Structure, relative to the structure_file.yaml. Or absolute from the repository root if a forward slash is used: `/requirements.txt`.
+run: # Defines the run-time configuration for the Structure
+  main_file: structure.py # Defines the path to the main Managed Structure Code, relative to the structure_file.yaml. Or absolute from the repository root if a forward slash is used: `/structure.py`.
+```
+
 ## Running Managed Code Outside of a Griptape Agent, Pipeline, or Workflow
+
```

Comment:
Do the spaces after the headers do anything?

Reply:
No, these were just added by my VS Code auto formatter.